app/models/knight.rb

```diff
@@ -1,3 +1,17 @@
 class Knight < Piece
 
+	def is_valid?(new_x, new_y)
+		total_x = new_x - self.x_coordinate

```

Comment:
You can also do it on one line like:  `total_x = (new_x - self.x_coordinate).abs`

Reply:
Just for the sake of readability and understanding: is there maybe a more descriptive name for total.. Sth like step_x or offset_x?